requirements.txt

```diff
@@ -7,9 +7,11 @@ bosdyn-mission==4.0.2
 grpcio==1.59.3
 image==1.5.33
 inflection==0.5.1
+opencv-python>=4.5.4
```

Comment:
Is there a way to not add this here? When you depend on opencv via rosdep in ros 1 or ros 2, it is pulled in via apt. We have had problems in the past with conflicting versions of opencv between apt and pip.

Reply:
@jbarry-bdai @khughes-bdai @tcappellari-bdai Thoughts? It should be a dependency for this to be a stand alone package, but then `spot_ros2` needs to uninstall it similar to what we do elsewhere which isn't great.

Reply:
I would maybe suggest just splitting up the requirements.txt between `spot_ros2` and `spot_wrapper`, and then using that to manage dependencies for both independently

Reply:
ya i like gary's suggestion of splitting them up (assuming that wouldn't break everything lol)